### ISSUE TITLE
httpx needs importing

### ISSUE THREAD

**Issue by sle78** [python-gitlab/python-gitlab#2984]
## Description of the problem, including code/CLI snippet
```python
File ~/PY310/lib/python3.10/site-packages/gitlab/client.py:1286, in GraphQL()
   1279 class GraphQL:
   1280     def __init__(
   1281         self,
   1282         url: Optional[str] = None,
   1283         *,
   1284         token: Optional[str] = None,
   1285         ssl_verify: Union[bool, str] = True,
-> 1286         client: Optional[httpx.Client] = None,
   1287         timeout: Optional[float] = None,
   1288         user_agent: str = gitlab.const.USER_AGENT,
   1289         fetch_schema_from_transport: bool = False,
   1290         max_retries: int = 10,
   1291         obey_rate_limit: bool = True,
   1292         retry_transient_errors: bool = False,
   1293     ) -> None:
   1294         if not _GQL_INSTALLED:
   1295             raise ImportError(
   1296                 "The GraphQL client could not be initialized because "
   1297                 "the gql dependencies are not installed. "
   1298                 "Install them with 'pip install python-gitlab[graphql]'"
   1299             )

NameError: name 'httpx' is not defined
```

## Specifications

  - python-gitlab version:
  - API version you are using (v3/v4):
  - Gitlab server version (or gitlab.com):

**Comment by JohnVillalovos** [comment]
I believe this has already been fixed in the latest release (4.11.1).

Previously reported issue: https://github.com/python-gitlab/python-gitlab/issues/2983

**Comment by JohnVillalovos** [comment]
I will mark this as closed. Re-open if you still have the issue when using the 4.11.1 release.
